.travis.yml

```diff
@@ -3,6 +3,6 @@ script:
   - sbt ++$TRAVIS_SCALA_VERSION test packageJS fullOptJS
```

Comment:
You will need to remove `packageJS` here. And actually it would be better to use:
```
sbt ++$TRAVIS_SCALA_VERSION 'set scalaJSStage in Global := FastOptStage' test 'set scalaJSStage in Global := FullOptStage' test
```